Interface/Harp.StepperDriver/Harp.StepperDriver.csproj

```diff
@@ -17,7 +17,7 @@
     <PackageLicenseFile>LICENSE</PackageLicenseFile>
     <PackageOutputPath>..\bin\$(Configuration)</PackageOutputPath>
     <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
-    <VersionPrefix>0.3.0-build2024032601</VersionPrefix>
+    <VersionPrefix>0.3.0-build240401</VersionPrefix>
```

Comment:
We really should agree on this at some point...

Reply:
@bruno-f-cruz yup, my convention right now is YYMMBB which I realize makes for a good balance between conciseness and rate of releases. The BB stands for "build number" in the month. So `240401` means the first build of April. I just realized over time that the exact day means very little to me here, you always have the commit for really fine precision if you need to, and having just a sense of the month is usually enough.